scripts/deploy-multus-network.sh

```diff
@@ -7,7 +7,6 @@ SCRIPT_DIR=$(dirname "$0")
 source "$SCRIPT_DIR"/init-env.sh
 
 MULTUS_GIT_URL="https://github.com/k8snetworkplumbingwg/multus-cni.git"
-WHEREABOUTS_GIT_URL="https://github.com/k8snetworkplumbingwg/whereabouts"
```

Comment:
Without wherebouts, the cni will assign ip addresses unique only per node. So without Whereabouts,  there will be cases where 2 different interfaces on 2 different nodes have the same IP